### DISCUSSION TITLE
Question about check_licenses_mapping.py

### DISCUSSION THREAD

**Discussion by shawnruby** [awslabs/scale-out-computing-on-aws#146] [General]
I am trying to implement the check_licenses_mapping hook.  I think I have everything right, but when I run it or  I look at the code, it seems it does not do anything.  It checks for the validity of the yaml file but then does nothing to check if the resource is available.  I assume it is supposed to run the command in the yaml and update the available resources and compare to the existing resources? It does not seem to do that and I don't see where in the code it should.
Am I missing something?

**Comment by mcrozes** [comment]
Hey @shawnruby -

Have you looked at https://awslabs.github.io/scale-out-computing-on-aws-documentation/tutorials/job-licenses-flexlm/ ? If yes, feel free to ping me directly on my github username at amazon dot com address to troubleshoot your issue.

**Comment by mcrozes** [comment]
Hello, closing this discussion as your question has been answered via email :)
